src/contexts/SignedInUserContext.jsx

```diff
@@ -51,8 +51,7 @@ export const SignedInUserContextProvider = ({ children }) => {
         });
         await Promise.all([
           updateUserActivity(session, podUrl),
-          createPublicContainer(session, podUrl),
-          createDocumentContainer(session, podUrl)
```

Comment:
Removing createDocumentContainer should be fine.

The new functionality for createDocument would have build the path to the file so it'll generate Document along the way when uploading any file for the first time and deleteDocument will only delete the specific container with the document in question inside it.

Reply:
correct